### PR TITLE
added snmp v3 user

### PR DESCRIPTION
**Pull request by nabhajit-ray**
### Description
[added snmp v3 users]

### Check List
- [ ] New functionality includes testing.
  - [x]  All tests pass for go 1.11 + gofmt checks.
- [ ] New functionality has been documented in the README if applicable.
  - [ ] New functionality has been thoroughly documented in the examples (please include helpful comments).
  - [ ] New endpoints supported are updated in the endpoints-support.md file.
- [ ] Changes are documented in the CHANGELOG.

### CODE REVIEW

**Review by VenkateshRavula — APPROVED** [review]

File: docs/d/appliance_snmpv3_user.html.markdown [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+# oneview\_appliance\_snmpv3\_user
+
+Use this data source to access the attributes of a snmpv3_user.
+
+## Example Usage
+
+```hcl
+data "oneview_appliance_snmpv3_user" "test" {
+ id_field = "67003649-af34-4a92-a46a-137855ddc8f7"
```
Comment (yuvirani) [inline docs/d/appliance_snmpv3_user.html.markdown:17]: Can we use Username instead of id field for Automation?

**Review by AsisBagga — APPROVED** [review]
LGTM

File: endpoints-support.md [hunk 2]
```diff
@@ -30,6 +30,11 @@
...
+|<sub>/rest/appliance/snmpv3-trap-forwarding/users/{id}</sub>                      |PUT       | :heavy_minus_sign:   | :heavy_minus_sign:   | :heavy_minus_sign:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:  | :white_check_mark:  |
+|<sub>/rest/appliance/snmpv3-trap-forwarding/users/{id}</sub>                      |DELETE    | :heavy_minus_sign:   | :heavy_minus_sign:   | :heavy_minus_sign:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:  | :white_check_mark:  |
```
Comment (yuvirani) [inline endpoints-support.md:37]: Is there any reason for not implementing Delete by username?

File: docs/d/appliance_snmpv3_user.html.markdown (hunk 1 above)
Comment (yuvirani) [reply docs/d/appliance_snmpv3_user.html.markdown:17]: Add example files also.

**Pull request merged**